plugins/microsoft-calendar/README.md

```diff
@@ -0,0 +1,13 @@
+# microsoft-calendar
```

Comment:
You'll want to update this with the proper content. Ideally a screenshot of the functionality so people can see what it does along with instructions as to how to install the plugin. Maybe look at this one for inspiration: https://github.com/backstage/backstage/tree/master/plugins/azure-devops

Reply:
I'd also include lots of details around the auth part of this, is there anything that needs to be setup before hand to make this work. Just mentioning this as it looks like we rely in the Microsoft Auth, which makes sense, but we need to make sure users of the plugin are aware of this